data_schema/variables/variable.json

```diff
@@ -4971,5 +4971,16 @@
               }
           ]
       }
+  ],
+  "zulip_api_keys":{
```

Comment:
**DO NOT** commit your real private info! Replace it with sample data, or just say:
```json
{
  "zulip_api_keys": {
    "email": "YOUR_ZULIP_EMAIL",
    "api_key": "YOUR_ZULIP_APIKEY"
  }
}
```